### PR TITLE
Bump ubi:8.1 to 8.3 to get latest security fixes

### PR DESCRIPTION
**Pull request by PushkarJ**
## Description
Bug fix: Redhat strongly recommends to upgrade to latest 8.3 ubi image to get the several (52 vulnerabilities for 25 packages) here: https://catalog.redhat.com/software/containers/ubi8/ubi-minimal/5c359a62bed8bd75a2c3fba8?tag=8.1&push_date=1586271046000&container-tabs=security

PS: this is my first contribution to calico, so happy to correct anything or follow any steps I may have missed

```release-note
Bump UBI from 8.1 to 8.3
```

### CODE REVIEW

**Review by caseydavenport — APPROVED** [review]
LGTM - thanks!

**Comment by marvin-tigera** [comment]
OK, I will merge the pull request when it's ready, leave the commits as is when I merge it, and leave the branch after I've merged it.

**Comment by PushkarJ** [comment]
@caseydavenport looks like some tests and builds failed. When I click on "Details" I get a 404 error. Did I miss something?

Also, the PR has docs and release notes required label. Should I be updating anything related to that?

**Comment by caseydavenport** [comment]
@PushkarJ looks good now! I added a release note and removed the docs-required label. Thanks!

**Comment by PushkarJ** [comment]
Thanks so much for this awesome contributor experience @caseydavenport :)

**Pull request merged**
